src/api/spec/models/project_spec.rb

```diff
@@ -326,4 +326,103 @@
       end
     end
   end
+
+  describe '#save_distributions' do
+    let(:factory) { create(:project, name: 'openSUSE:Factory') }
```

Comment:
Can be better to use `factory_project` instead of just `factory`? I mean... that's somehow misleading because we use factories from FactoryGirl... and call a factory `factory` sounds tricky :D